src/components/Shared/SelectedRevisionsTable.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+
+import Close from '@mui/icons-material/Close';
+import IconButton from '@mui/material/IconButton';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import { connect, useDispatch } from 'react-redux';
+
+import { deleteRevision } from '../../reducers/SelectedRevisions';
+import type { TableHeaders } from '../../types/enums';
+import { Revision, State } from '../../types/state';
+
+const tableHeaderDetails: TableHeaders[] = [
+  'Project',
+  'Revision',
+  'Author',
+  'Commit Message',
+  'Timestamp',
+];
+
+function SelectedRevisionsTable(props: SelectedRevisionsProps) {
+  const { revisions } = props;
+  const dispatch = useDispatch();
+
+  if (revisions.length === 0) {
+    return <div className="missingRevisions">Add some revisions</div>;
+  }
+  return (
+    <TableContainer className="layout">
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell component="th" scope="row" />
+            {tableHeaderDetails.map((header: TableHeaders) => (
+              <TableCell key={header} sx={{ fontWeight: 600 }}>
+                {header}
+              </TableCell>
+            ))}
+            <TableCell />
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {revisions.map((row: Revision, index: number) => (
+            <TableRow key={row.id}>
+              <TableCell
+                sx={{
+                  '.cellStyle': {
+                    fontWeight: 600,
+                    borderRadius: '2.5px',
+                    padding: '5px',
+                    display: 'flex',
+                    color: '#737373',
+                    width: '50px',
+                    border: 'solid 1px #737373',
+                  },
```

Comment:
I think we could move this into the `componets.js` MuiTableCell right after the `paddingBottom` at line 15. Inline CSS is not a recommended approach.

Reply:
thanks!